workshops/geometric_pattern/README.md

```diff
@@ -1,15 +1,26 @@
 # Geometric Pattern
 
-![](img/sample1.png)
+_**We recommend going through this workshop in Google Chrome.**_
 
-Short link to this workshop: https://workshops.hackclub.com/geometric_pattern
+![](img/sample1.png)
 
 [Demo](http://prophetorpheus.github.io/geometric_pattern/)
 
+Here's the [live demo][final_live_demo].
```

Comment:
What is the difference between the "live" demo and the non live demo - They seem to link to the same spot.